src/Col.js

```diff
@@ -78,6 +78,10 @@ const Col = (props) => {
     }
   });
 
+  if (attributes['xs']) {
```

Comment:
I'd be happy to add back support for `false` as I think `Col` should have some sort of default that takes care of the typical use case. I recently and foolishly moved `delete attributes[colWidth];` after the prop check, it should instead revert back to the following in `colWidths.forEach(colWidth => {`:

```
// always clear out width props from attributes
delete attributes[colWidth];

if (!columnProp) {
  return;
}
```